docs/website/versioned_docs/version-3.0.0/command-reference/dev.md

```diff
@@ -0,0 +1,226 @@
+---
+title: odo dev
+sidebar_position: 2
+---
+
+## Description
+
+`odo dev` is used in order to quickly and effectively iterate through your development process for building an application.
+
+This is [inner loop](http://localhost:3000/docs/3.0.0/intro#what-is-inner-loop-and-outer-loop) development and allows you to code, build, run and test the application in a continuous workflow.
```

Comment:
Looks like this should not link to `localhost:3000`.

Reply:
Woops! Good catch.